data/packrat/artifacts.json

```diff
@@ -0,0 +1,2052 @@
+        "credential_type": "sqlite",
+        "sql_search": [
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "urls",
+            "sql_column": "url"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "keyword_search_terms",
+            "sql_column": "lower_term"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "downloads",
+            "sql_column": "current_path, tab_referrer_url"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "segments",
+            "sql_column": "name"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "downloads_url_chains",
+            "sql_column": "url"
+          }
+        ]
+      },
+      {
+        "filetypes": "web_history",
+        "path": "LocalAppData",
+        "dir": "COMODO",
+        "artifact": "Visited Links",
+        "description": "Comodo's History",
+        "credential_type": "sqlite",
+        "sql_search": [
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "urls",
+            "sql_column": "url"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "keyword_search_terms",
+            "sql_column": "lower_term"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "downloads",
+            "sql_column": "current_path, tab_referrer_url"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "segments",
+            "sql_column": "name"
+          },
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "downloads_url_chains",
+            "sql_column": "url"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "coolnovo",
+    "category": "browsers",
+    "file_artifact": [
+      {
+        "filetypes": "logins",
+        "path": "LocalAppData",
+        "dir": "MapleStudio",
+        "artifact": "Login Data",
+        "description": "CoolNovo saved Username and Passwords",
+        "credential_type": "sqlite",
+        "sql_search": [
+          {
+            "sql_description": "Database Commands which exports Chrome's Login data",
+            "sql_table": "logins",
+            "sql_column": "action_url, username_value"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "firefox",

```

Comment:
maybe its time to convert this to a post module? https://github.com/rapid7/metasploit-framework/blob/76954957c740525cff2db5a60bcf936b4ee06c42/scripts/meterpreter/enum_firefox.rb